### ISSUE TITLE
The opponents are not always displayed

### ISSUE THREAD

**Issue by antiproton** [jsonchess/lightsquare#102]
The opponents are not always displayed. I open two browser windows Firefox & Chrome and start a game. The enemy is not displayed in the second window. Best Regards R.U.

![lightsquare](https://cloud.githubusercontent.com/assets/6153753/5796577/d897c472-9faa-11e4-8e54-f3e1060a5048.jpg)

**Comment by gushogg-blake** [comment]
This is probably because the player on the left's rating is outside the accepted range of the player on the right - if you try it with a bigger range you should see it.

**Comment by antiproton** [comment]
Yes true, that is a little complicated for me ... The Elo number should be visible on the initial screen. Maybe at first no ELO restriction. This brings more visitors. The timing is very good!
Best Regards R.U.

**Comment by gushogg-blake** [comment]
I agree, it could do to be clearer.  Thanks - I think I'm somewhere around 1600 on chess.com but I don't have an official rating
